src/js/model/string/string.js

```diff
@@ -0,0 +1,432 @@
+	    return "<";
+	},
+	"3D": function() {
+	    return "=";
+	},
+	"3E": function() {
+	    return ">";
+	},
+	"3F": function() {
+	    return "?";
+	},
+	"40": function() {
+	    return "@";
+	},
+	"41": function() {
+	    return "A";
+	},
+	"42": function() {
+	    return "B";
+	},
+	"43": function() {
+	    return "C";
+	},
+	"44": function() {
+	    return "D";
+	},
+	"45": function() {
+	    return "E";
+	},
+	"46": function() {
+	    return "F";
+	},
+	"47": function() {
+	    return "G";
+	},
+	"48": function() {
+	    return "H";
+	},
+	"49": function() {
+	    return "I";
+	},
+	"4A": function() {
+	    return "J";
+	},
+	"4B": function() {
+	    return "K";
+	},
+	"4C": function() {
+	    return "L";
+	},
+	"4D": function() {
+	    return "M";
+	},
+	"4E": function() {
+	    return "N";
+	},
+	"4F": function() {
+	    return "O";
+	},
+	"50": function() {
+	    return "P";
+	},
+	"51": function() {
+	    return "Q";
+	},
+	"52": function() {
+	    return "R";
+	},
+	"53": function() {
+	    return "S";
+	},
+	"54": function() {
+	    return "T";
+	},
+	"55": function() {
+	    return "U";
+	},
+	"56": function() {
+	    return "V";
+	},
+	"57": function() {
+	    return "W";
+	},
+	"58": function() {
+	    return "X";
+	},
+	"59": function() {
+	    return "Y";
+	},
+	"5A": function() {
+	    return "Z";
+	},
+	"5B": function() {
+	    return "[";
+	},
+	"5C": function() {
+	    return "\\";
+	},
+	"5D": function() {
+	    return "]";
+	},
+	"5E": function() {
+	    return "^";
+	},
+	"5F": function() {
+	    return "_";
+	},
+	"60": function() {
+	    return "`";
+	},
+	"61": function() {
+	    return "a";
+	},
+	"62": function() {
+	    return "b";
+	},
+	"63": function() {
+	    return "c";
+	},
+	"64": function() {
+	    return "d";
+	},
+	"65": function() {
+	    return "e";
+	},
+	"66": function() {
+	    return "f";
+	},
+	"67": function() {
+	    return "g";
+	},
+	"68": function() {
+	    return "h";
+	},
+	"69": function() {
+	    return "i";
+	},
+	"6A": function() {
+	    return "j";
+	},
+	"6B": function() {
+	    return "k";
+	},
+	"6C": function() {
+	    return "l";
+	},
+	"6D": function() {
+	    return "m";
+	},
+	"6E": function() {
+	    return "n";
+	},
+	"6F": function() {
+	    return "o";
+	},
+	"70": function() {
+	    return "p";
+	},
+	"71": function() {
+	    return "q";
+	},
+	"72": function() {
+	    return "r";
+	},
+	"73": function() {
+	    return "s";
+	},
+	"74": function() {
+	    return "t";
+	},
+	"75": function() {
+	    return "u";
+	},
+	"76": function() {
+	    return "v";
+	},
+	"77": function() {
+	    return "w";
+	},
+	"78": function() {
+	    return "x";
+	},
+	"79": function() {
+	    return "y";
+	},
+	"7A": function() {
+	    return "z";
+	},
+	"7B": function() {
+	    return "{";
+	},
+	"7C": function() {
+	    return "|";
+	},
+	"7D": function() {
+	    return "}";
+	},
+	"7E": function() {
+	    return "~";
+	},
+	"7F": function() {
+	    return "<DEL>";
+	}
+    }
```

Comment:
You can get rid of this whole lookup table by passing the return value of [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) into [String.fromCharCode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode)